openedx/core/djangoapps/courseware_api/urls.py

```diff
@@ -7,11 +7,16 @@
 from django.conf.urls import url
 
 from openedx.core.djangoapps.courseware_api import views
+from openedx.core.djangoapps.content.learning_sequences.api import USAGE_KEY_HASH_PATTERN
+
 
 urlpatterns = [
     url(fr'^course/{settings.COURSE_KEY_PATTERN}',
         views.CoursewareInformation.as_view(),
         name="courseware-api"),
+    url(fr'^sequence/{USAGE_KEY_HASH_PATTERN}',
+        views.SequenceMetadataByHash.as_view(),
+        name="sequence-api-by-hash"),
```

Comment:
Ah. This is totally fine, but FWIW, I had meant referencing the exact same view, but sending different kwargs to it so that the code could differentiate whether it was being called by usage key or hash:

https://docs.djangoproject.com/en/2.2/topics/http/urls/#views-extra-options

Reply:
Ah! I think I will leave it, but that is a great TIL.